### PR TITLE
Update log4rs for tari_stratum_transcoder

### PR DESCRIPTION
**Pull request by StriderDM**
## Description
Update log4rs for tari_stratum_transcoder

## Motivation and Context
Separate tari_stratum_transcoder logs from tari_merge_mining_proxy  logs.

## Checklist:
* [x] I'm merging against the `development` branch.
* [x] I have squashed my commits into a single commit.

### CODE REVIEW

File: common/logging/log4rs_sample_transcoder.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# A sample log configuration file for running in release mode. By default, this configuration splits up log messages to
+# three destinations:
+#    * Console: For log messages with level INFO and higher
+#    * log/proxy/proxy.log: All proxy logs will be written to this file
```
Comment (hansieodendaal) [inline common/logging/log4rs_sample_transcoder.yml]: maybe `log/transcoder/transcoder.log?`

File: common/logging/log4rs_sample_transcoder.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+#
+#  See https://docs.rs/log4rs/0.8.3/log4rs/encode/pattern/index.html for deciphering the log pattern. The log format
+#  used in this sample configuration prints messages as:
+#  timestamp [target] LEVEL message
+refresh_rate: 30 seconds
+appenders:
+  # An appender named "proxy" that writes to a file with a custom pattern encoder
+  proxy:
```
Comment (hansieodendaal) [inline common/logging/log4rs_sample_transcoder.yml]: Maybe `transcoder` ?

File: common/logging/log4rs_sample_transcoder.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        count: 50
+        pattern: "log/transcoder/transcoder.{}.log"
+    encoder:
+      pattern: "{d(%Y-%m-%d %H:%M:%S.%f)} [{t}] {l:5} {m}{n}"
+
+# root (to proxy)
+root:
+  level: debug
+  appenders:
+    - proxy
```
Comment (hansieodendaal) [inline common/logging/log4rs_sample_transcoder.yml]: Maybe `transcoder` ?

File: common/logging/log4rs_sample_transcoder.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        limit: 200mb
+      roller:
+        kind: fixed_window
+        base: 1
+        count: 50
+        pattern: "log/transcoder/transcoder.{}.log"
+    encoder:
+      pattern: "{d(%Y-%m-%d %H:%M:%S.%f)} [{t}] {l:5} {m}{n}"
+
+# root (to proxy)
```
Comment (hansieodendaal) [inline common/logging/log4rs_sample_transcoder.yml]: Maybe `transcoder` ?

File: common/logging/log4rs_sample_transcoder.yml (hunk 2 above)
Comment (StriderDM) [reply common/logging/log4rs_sample_transcoder.yml]: Thanks for picking up  the typos 👍🏻

**Review by stringhandler — APPROVED** [review]

**Pull request merged**